src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,72 @@
+.menu {
+  box-sizing: border-box;
```

Comment:
checklist #17
Fix menu for small screens (if there is not enough space for all the menu items)

Reply:
fixed